tests/e2e-tests/scripts/run-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+if [[ ${RUN_E2E} == 1 ]]; then
+	set -ev
+	# Setup
+	export DISPLAY=:99.0
+	sh -e /etc/init.d/xvfb start
+ 	sleep 3
+	npm install
+	export NODE_CONFIG_DIR="./tests/e2e-tests/config"
```

Comment:
This is probably a Node beginner question and not directly related to your PR, but, in case you know the answer, I'm curious to know why we need to declare `NODE_CONFIG_DIR` everywhere and I wonder if there is a way to do it only once and avoid repetition?